x/stakeibc/keeper/msg_server_submit_tx.go

```diff
@@ -206,3 +165,24 @@ func (k Keeper) SubmitTxs(ctx sdk.Context, connectionId string, msgs []sdk.Msg,
 
 	return nil
 }
+
+func (k Keeper) GetLightClientHeightSafely(ctx sdk.Context, connectionID string) (int64, bool) {
```

Comment:
do we need this?

Reply:
unused currently, it updated the LC to latest height,  helpful to check HZ attributes